msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md

```diff
@@ -1,33 +1,36 @@
 ---
```

Comment:
Same comments as given for invoking-task-modules.

Reply:
Awaiting inputs from Wajeed